### DISCUSSION TITLE
MAPBOX_API_KEY  CONFIG Use a public access token (pk.*) with Mapbox GL, not a secret access token (sk.*)

### DISCUSSION THREAD

**Discussion by Lubyam** [apache/superset#20041] [Q&A / Help] [answered]
I want to use deck.gl Scatterplot and set the MAPBOX_API_KEY   with "sk.eyJ1IjoieGFuaWN1IiwiYSI6ImNsMnp2aW9qbjFqaTYzaX********",
![image](https://user-images.githubusercontent.com/7630366/167970047-8c44b2bc-c792-490f-b255-58a25b4a1572.png)

when i update the chart got a error info:

![image](https://user-images.githubusercontent.com/7630366/167969991-ebafa15b-4be8-4e62-9af8-af9f41124b6f.png)

How should I solve it？

**Comment by stephenLYZ** [comment, marked as answer]
Hi @Lubyam You need to use public token instead of secret token, which is always prefixed with `pk`. Please refer to the url from the error message.

**Comment by Lubyam** [comment]
> Hi @Lubyam You need to use public token instead of secret token, which is always prefixed with `pk`. Please refer to the url from the error message.

@stephenLYZ  thank you 
Turns out I had it backwards all along
